### PR TITLE
Update office-365-groups-ww.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated reports in the security and compliance center link

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

**Comment by msbemba** [comment]
Issue#https://github.com/MicrosoftDocs/microsoft-365-docs/issues/7093

File: microsoft-365/admin/activity-reports/office-365-groups-ww.md [hunk 1]
```diff
@@ -70,6 +70,6 @@ The **groups** report can be viewed for trends over the last 7 days, 30 days, 90
 ## Related content
 
 [Microsoft 365 Reports in the admin center](activity-reports.md) (article)\
-[Reports in the Security & Compliance Center](../../compliance/reports-in-security-and-compliance.md) (article)\
+[Reports in the Security & Compliance Center](microsoft-365/security/office-365-security/reports-and-insights-in-security-and-compliance) (article)\
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/office-365-groups-ww.md]:
```suggestion
[Smart reports and insights in the Security & Compliance Center](/microsoft-365/security/office-365-security/reports-and-insights-in-security-and-compliance) (article)\
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@kwekuako Please merge this PR. Thanks

**Pull request merged**